xcube/core/store/ref/store.py

```diff
@@ -0,0 +1,186 @@
+    :param target_protocol: Used for loading the reference files.
+        If not given, protocol will be derived from the given path.
+    :param target_options: Extra filesystem options for loading
+        the reference files.
+    :param remote_protocol: The protocol of the filesystem on which
+        the references will be evaluated. If not given, will be derived
+        from the first URL in the references that has a protocol.
+    :param remote_options: Extra filesystem options for loading the
+        referenced data.
+    :param max_gap: See ``max_block``.
+    :param max_block: For merging multiple concurrent requests to the same
+        remote file. Neighboring byte ranges will only be
+        merged when their inter-range gap is <= `max_gap`.
+        Default is 64KB. Set to 0 to only merge when it
+        requires no extra bytes. Pass a negative number to
+        disable merging, appropriate for local target files.
+        Neighboring byte ranges will only be merged when the
+        size of the aggregated range is <= ``max_block``.
+        Default is 256MB.
+    :param cache_size: Maximum size of LRU cache, where
+        cache_size*record_size denotes the total number of
+        references that can be loaded in memory at once.
+        Only used for lazily loaded references.
+    """
+
+    def __init__(
+            self,
+            refs: List[str],
+            **ref_kwargs,
+    ):
+        self._refs = dict(self._normalize_ref(ref) for ref in refs)
+        self._ref_kwargs = ref_kwargs
+
+    @classmethod
+    def get_data_store_params_schema(cls) -> JsonObjectSchema:
+        return REF_STORE_SCHEMA
+
+    @classmethod
+    def get_data_types(cls) -> Tuple[str, ...]:
+        return ("dataset",)
+
+    def get_data_types_for_data(self, data_id: str) -> Tuple[str, ...]:
+        return self.get_data_types()
+
+    def get_data_ids(self,
+                     data_type: DataTypeLike = None,
+                     include_attrs: Container[str] = None) -> \
+            Union[Iterator[str], Iterator[Tuple[str, Dict[str, Any]]]]:
+        return iter(self._refs.keys())
+
+    def has_data(self,
+                 data_id: str,
+                 data_type: DataTypeLike = None) -> bool:
+        return data_id in self._refs
+
+    def describe_data(self,
+                      data_id: str,
+                      data_type: DataTypeLike = None) -> DataDescriptor:
+        data_descriptor = self._refs[data_id].get("data_descriptor")
+        if data_descriptor is None:
+            dataset = self.open_data(data_id)
+            data_descriptor = new_data_descriptor(data_id, dataset)
+            self._refs[data_id]["data_descriptor"] = data_descriptor
+        return data_descriptor
+
+    def get_data_opener_ids(self,
+                            data_id: str = None,
+                            data_type: DataTypeLike = None) -> Tuple[str, ...]:
+        return ("dataset:zarr:reference",)
```

Comment:
```suggestion
        return "dataset:zarr:reference",
```

Reply:
I disagree. This is intentional to make clear its a tuple and not a misplaced comma.